README.md

```diff
@@ -390,6 +391,37 @@ end
 
 See more examples of pairs queries [here.](https://github.com/tarantool/crud/blob/master/doc/pairs.md)
 
+### Cut_rows 
```

Comment:
`Cut extra rows`? I think that it would be good to add a note that this function is useful with `fields` parameter.